Update ru_ru.json

pgkrol opened the pull request:
Updating and fixing bugs

pgkrol committed:
Update ru_ru.json

Updating and fixing bugs

Hanqnero reviewed (changes requested):
1. You used `"` inside strings and didn't escaped them with `\`, that breakes whole json structure.
2. Мне кажется, что в названия штуковин **на** вагонетках стоит добавить этот предлог, то есть вместо `Штуковина грузовой вагонетки` будет `Штуковина **на** грузовой вагонетке` и тому подобное, потому что сама конструкция стоит на вагонетке.
3. [Line 654] Curiosities ≠ курьёзы.
4. [Lines 808-816] 
4.1 Когда речь идёт про узор/порядок, то его просто называют шахматным, доска тут не при чём. 
4.2 Лучше звучит не перевёрнутый, а обратный, так как речь идёт про узор, порядок размещения.
4.3 При чём тут **крен**?! Речь идёт про шанс установки блока.
4.4 Шаблон Solid, конечно можно перевести как "всё", но как вам вариант "сплошной"?
5. Почти везде на словах, пишущихся с заглавной буквы ставился смысловой акцент, на языке оригинала они тоже писались с заглавной буквы, поэтому я их и сохранил.

pgkrol committed:
Update ru_ru.json

Added Hanqnero's fixes (Ru localization update #1974)

pgkrol commented:
Added Hanqnero's fixes (Ru localization update #1974)

Hanqnero commented on src/main/resources/assets/create/lang/ru_ru.json:
```diff
@@ -2200,22 +2200,22 @@
-	"create.ponder.spout_filling.text_5": "Когда предметы доставлены по конвейеру...",
+	"create.ponder.spout_filling.text_4": "Наполняемые предметы могут быть размещены на депо под дозатором",
+	"create.ponder.spout_filling.text_5": "Когда предметы подаются конвейером...",
 	"create.ponder.spout_filling.text_6": "Дозатор будет удерживать и обрабатывать их автоматически,
```
Пропущена закрывающая строку кавычка
Missing a closing quote

pgkrol replied:
Спасибо!

pgkrol committed:
Fixed translation errors, thanks to Hanqnero.

Fixed translation errors, thanks to Hanqnero. And some correction of spelling and punctuation errors.

pgkrol commented:
@Hanqnero, добрый день.

_21 Не согласен с этим фиксом, ведь утилизация - это процесс, а эта строка показывается при наведении на слот результата в процессе крафта, то есть на предмет: можно получить желаемый предмет, а можно получить случайный мусор с некоторым шансом. Посмотрите, пожалуйства, как это в игре выглядит, может вы со мной согласитесь._

Исходил из того что в https://github.com/Creators-of-Create/Create/commit/2300e0e58a0288fe7bd1dca6be1ccaae71eba9e2
сменили junk на salvage, соотнес это со свалками машин (salvage yard) - заменю на ваш перевод.

_39 Орфографическая ошибка в слове._
Исправил, спасибо. Так же постарался исправить все ошибки встречаемые в файле.

Hanqnero commented:
Спасибо вам за принятие моих правок, за проделанную работу в целом.

simibubi commented:
Many thanks from the team as well 👍
